src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -227,6 +212,74 @@ public DataAccessRequest createDataAccessRequest(User user, DataAccessRequest da
     return findByReferenceId(referenceId);
   }
 
+  /**
+   * Create a progress report for the given DataAccessRequest.
+   * The parent DAR is just passed in for validation purposes.
+   *
+   * @param user              The User
+   * @param progressReport    The DataAccessRequest
+   * @param parentDar         The parent DataAccessRequest
+   * @return The created progress report.
+   */
+  public DataAccessRequest createProgressReport(User user, DataAccessRequest progressReport, DataAccessRequest parentDar) {
+    validateProgressReport(user, progressReport, parentDar);
+
+    Date now = new Date();
+    String referenceId = progressReport.getReferenceId();
+    List<Integer> datasetIds = progressReport.getDatasetIds();
+    dataAccessRequestDAO.insertProgressReport(
+          Integer.valueOf(progressReport.getParentId()),
+          progressReport.getCollectionId(),
+          referenceId,
+          user.getUserId(),
+          now,
+          now,
+          now,
+          now,
+          progressReport.getData());
+    syncDataAccessRequestDatasets(datasetIds, referenceId);
```

Comment:
Not for this PR - it's definitely scope creep. The insert and sync should really be in a single transaction-level method. This also applies to any call where we're modifying a DAR - draft updates, for example.